docs/_build/html/_sources/docker_compose.md.txt

```diff
@@ -0,0 +1,22 @@
+# Docker Compose
+Tilt supports `docker-compose` as a runtime, which makes it easy to get Tilt's better UX without changing your config or tools. This doc describes how you can get started with a one-line config, which is even simpler than the [Tutorial](tutorial.html) for Kubernetes projects.
```

Comment:
> as a runtime

while this may be technically correct, i suspect it'll confuse some portion of people reading this doc... maybe:

> Tilt can run `docker-compose` under the hood to orchestrate your services

Reply:
How's this?